binary_search_tree/tree.py

```diff
@@ -7,53 +7,118 @@ def __init__(self, key, val = None):
         self.value = val
         self.left = None
         self.right = None
-        
+
+    def height(self):
```

Comment:
Why is this up here?

Reply:
I was having a hard time wrapping my head around starting from a root and working my way down so it's essentially a helper function to get the height of an individual node and using that to determine the overall tree height. Is this a bad way to do this?